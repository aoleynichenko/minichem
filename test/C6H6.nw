# C6H6 single-point energy
# HF molecular orbitals will be written to molden-format file c6h6.mos

start C6H6
memory 10 mb
echo
#nproc 8

geometry
C  0.000  1.396  0.000
C  1.209  0.698  0.000
C  1.209 -0.698  0.000
C  0.000 -1.396  0.000
C -1.209 -0.698  0.000
C -1.209  0.698  0.000
H  0.000  2.479  0.000
H  2.147  1.240  0.000
H  2.147 -1.240  0.000
H  0.000 -2.479  0.000
H -2.147 -1.240  0.000
H -2.147  1.240  0.000
end

basis "sto-3g"
H    S
      3.42525091             0.15432897       
      0.62391373             0.53532814       
      0.16885540             0.44463454  
C    S
     71.6168370              0.15432897       
     13.0450960              0.53532814       
      3.5305122              0.44463454       
C    S
      2.9412494             -0.09996723
      0.6834831              0.39951283    
      0.2222899              0.70011547            
C    P
      2.9412494              0.15591627       
      0.6834831              0.60768372       
      0.2222899              0.39195739    
end

#scf
#  print "overlap"
 # diis 5
#end

#out
#  molden
#end

task scf
