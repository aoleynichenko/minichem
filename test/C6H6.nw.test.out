Minichem 1.0
By Alexander Oleynichenko, 2018
Build date:    Jan  2 2019 22:39:09
Compiler:      gcc 4.9.2
Date:          Fri Jan  4 14:00:04 2019
# MPI threads: 1
Arguments:  minichem.x C6H6.nw 
------------------------------------------------------------
>>> C6H6.nw
# C6H6 single-point energy
# HF molecular orbitals will be written to molden-format file c6h6.mos

start C6H6
memory 10 mb
echo
#nproc 8

geometry
C  0.000  1.396  0.000
C  1.209  0.698  0.000
C  1.209 -0.698  0.000
C  0.000 -1.396  0.000
C -1.209 -0.698  0.000
C -1.209  0.698  0.000
H  0.000  2.479  0.000
H  2.147  1.240  0.000
H  2.147 -1.240  0.000
H  0.000 -2.479  0.000
H -2.147 -1.240  0.000
H -2.147  1.240  0.000
end

basis "sto-3g"
H    S
      3.42525091             0.15432897       
      0.62391373             0.53532814       
      0.16885540             0.44463454  
C    S
     71.6168370              0.15432897       
     13.0450960              0.53532814       
      3.5305122              0.44463454       
C    S
      2.9412494             -0.09996723
      0.6834831              0.39951283    
      0.2222899              0.70011547            
C    P
      2.9412494              0.15591627       
      0.6834831              0.60768372       
      0.2222899              0.39195739    
end

#scf
#  print "overlap"
 # diis 5
#end

out
  molden
end

task scf
------------------------------------------------------------
                    Basis information
   H
  ----
  1   1S     3.42525091     0.15432897
             0.62391373     0.53532814
             0.16885540     0.44463454


   C
  ----
  1   1S    71.61683700     0.15432897
            13.04509600     0.53532814
             3.53051220     0.44463454

  2   2S     2.94124940    -0.09996723
             0.68348310     0.39951283
             0.22228990     0.70011547

  3   2P     2.94124940     0.15591627
             0.68348310     0.60768372
             0.22228990     0.39195739


Cartesian basis set summary:
*---------*-----------------*--------*------------*
| Element |   Description   | Shells | Primitives |
*---------*-----------------*--------*------------*
|   H     | sto-3g          |   1    |     3      |
|   C     | sto-3g          |   3    |     9      |
*---------*-----------------*--------*------------*

------------------------------------------------------------

		AO integrals evaluation module
		------------------------------

Output files:
  AOINTS1   one-electron integrals
  AOINTS2   two-electron integrals
One-electron integrals evaluation algorithm: Obara-Saika
Two-electron integrals evaluation algorithm: Obara-Saika

	Atomic coordinates (angstroms):
    [ 1]  C      0.00000000     1.39599990     0.00000000
    [ 2]  C      1.20899991     0.69799995     0.00000000
    [ 3]  C      1.20899991    -0.69799995     0.00000000
    [ 4]  C      0.00000000    -1.39599990     0.00000000
    [ 5]  C     -1.20899991    -0.69799995     0.00000000
    [ 6]  C     -1.20899991     0.69799995     0.00000000
    [ 7]  H      0.00000000     2.47899982     0.00000000
    [ 8]  H      2.14699985     1.23999991     0.00000000
    [ 9]  H      2.14699985    -1.23999991     0.00000000
    [10]  H      0.00000000    -2.47899982     0.00000000
    [11]  H     -2.14699985    -1.23999991     0.00000000
    [12]  H     -2.14699985     1.23999991     0.00000000


	Distance matrix (angstroms):
    [ 1] C     [ 2] C      1.39602461
    [ 1] C     [ 3] C      2.41795702
    [ 1] C     [ 4] C      2.79199980
    [ 1] C     [ 5] C      2.41795702
    [ 1] C     [ 6] C      1.39602461
    [ 1] C     [ 7] H      1.08299992
    [ 1] C     [ 8] H      2.15265983
    [ 1] C     [ 9] H      3.39972107
    [ 1] C     [10] H      3.87499972
    [ 1] C     [11] H      3.39972107
    [ 1] C     [12] H      2.15265983
    [ 2] C     [ 3] C      1.39599990
    [ 2] C     [ 4] C      2.41795702
    [ 2] C     [ 5] C      2.79204923
    [ 2] C     [ 6] C      2.41799983
    [ 2] C     [ 7] H      2.15258945
    [ 2] C     [ 8] H      1.08333182
    [ 2] C     [ 9] H      2.15306464
    [ 2] C     [10] H      3.39926585
    [ 2] C     [11] H      3.87538099
    [ 2] C     [12] H      3.39948501
    [ 3] C     [ 4] C      1.39602461
    [ 3] C     [ 5] C      2.41799983
    [ 3] C     [ 6] C      2.79204923
    [ 3] C     [ 7] H      3.39926585
    [ 3] C     [ 8] H      2.15306464
    [ 3] C     [ 9] H      1.08333182
    [ 3] C     [10] H      2.15258945
    [ 3] C     [11] H      3.39948501
    [ 3] C     [12] H      3.87538099
    [ 4] C     [ 5] C      1.39602461
    [ 4] C     [ 6] C      2.41795702
    [ 4] C     [ 7] H      3.87499972
    [ 4] C     [ 8] H      3.39972107
    [ 4] C     [ 9] H      2.15265983
    [ 4] C     [10] H      1.08299992
    [ 4] C     [11] H      2.15265983
    [ 4] C     [12] H      3.39972107
    [ 5] C     [ 6] C      1.39599990
    [ 5] C     [ 7] H      3.39926585
    [ 5] C     [ 8] H      3.87538099
    [ 5] C     [ 9] H      3.39948501
    [ 5] C     [10] H      2.15258945
    [ 5] C     [11] H      1.08333182
    [ 5] C     [12] H      2.15306464
    [ 6] C     [ 7] H      2.15258945
    [ 6] C     [ 8] H      3.39948501
    [ 6] C     [ 9] H      3.87538099
    [ 6] C     [10] H      3.39926585
    [ 6] C     [11] H      2.15306464
    [ 6] C     [12] H      1.08333182
    [ 7] H     [ 8] H      2.47885641
    [ 7] H     [ 9] H      4.29424817
    [ 7] H     [10] H      4.95799964
    [ 7] H     [11] H      4.29424817
    [ 7] H     [12] H      2.47885641
    [ 8] H     [ 9] H      2.47999982
    [ 8] H     [10] H      4.29424817
    [ 8] H     [11] H      4.95871278
    [ 8] H     [12] H      4.29399969
    [ 9] H     [10] H      2.47885641
    [ 9] H     [11] H      4.29399969
    [ 9] H     [12] H      4.95871278
    [10] H     [11] H      2.47885641
    [10] H     [12] H      4.29424817
    [11] H     [12] H      2.47999982

generating atom-centered basis set...
  # bfns   = 36
  # shells = 24

	Basis functions (atomic units):
    C    0.000   2.638   0.000  [1]s  [2]s  [3]pz  [4]py  [5]px
    C    2.285   1.319   0.000  [6]s  [7]s  [8]pz  [9]py  [10]px
    C    2.285  -1.319   0.000  [11]s  [12]s  [13]pz  [14]py  [15]px
    C    0.000  -2.638   0.000  [16]s  [17]s  [18]pz  [19]py  [20]px
    C   -2.285  -1.319   0.000  [21]s  [22]s  [23]pz  [24]py  [25]px
    C   -2.285   1.319   0.000  [26]s  [27]s  [28]pz  [29]py  [30]px
    H    0.000   4.685   0.000  [31]s
    H    4.057   2.343   0.000  [32]s
    H    4.057  -2.343   0.000  [33]s
    H    0.000  -4.685   0.000  [34]s
    H   -4.057  -2.343   0.000  [35]s
    H   -4.057   2.343   0.000  [36]s

begin 1e integrals...
  overlap
  kinetic energy
  nuclear attraction
  X Y Z integrals
  XX XY XZ YY YZ ZZ integrals
    center: origin = 0.0 0.0 0.0
done
begin 2e integrals...
  sizeof buf (bytes) = 98304
  sizeof integral (bytes) = 24
  # unique ERIs   = 222111
  # non-zero ERIs = 127945
done

------------------------------------------------------------

          ********************************
          *      Parallel SCF Module     *
          ********************************

                    SCF Parameters
                    --------------
               print level : medium
              wavefunction : RHF
            max iterations : 50
              density conv : 1e-06
               energy conv : 1e-07
           integral direct : off
                      diis : on
                   diisbas : 5


Atoms:     12
Electrons: 0
Mol mass:  0.000
Charge:    0
Spin mult: 1
Units:     angstroms
One-electron integrals read in 0.000000 sec
Basis orthogonalization algorithm: canonical
Basis functions elimination threshold: 1e-08
Overlap matrix lowest eigenvalue = 0.17195555
AO basis orthogonalization done in 0.091304 sec

Initial guess: extended Huckel theory
Initial energy = -429.76043481

 iter.       Energy         Delta E       RMS-Dens       DIIS-Err     time
----------------------------------------------------------------------------
   1    -229.31830420    -2.92583491     0.91076057     0.30543339    0.17
   2    -227.58718307     1.73112113     0.24353524     0.04234603    0.19
   3    -227.93799952    -0.35081645     0.04430170     0.00598920    0.20
   4    -227.88916423     0.04883529     0.00352539     0.00040233    0.22
   5    -227.89088132    -0.00171709     0.00030488     0.00002394    0.23
   6    -227.89074306     0.00013826     0.00001088     0.00000195    0.25
   7    -227.89074407    -0.00000100     0.00000200     0.00000027    0.26
   8    -227.89074473    -0.00000066     0.00000024     0.00000010    0.28
----------------------------------------------------------------------------
          Total SCF energy =  -227.89074473
  Nuclear repulsion energy =   203.36796552



         Molecular Orbitals Summary
  +-----+-----+----------------+----------+
  | No  | Occ |     Energy     | Symmetry |
  +-----+-----+----------------+----------+
  | 1   |  2  |   -11.02908708 |     ?    |
  | 2   |  2  |   -11.02907187 |     ?    |
  | 3   |  2  |   -11.02905315 |     ?    |
  | 4   |  2  |   -11.02873573 |     ?    |
  | 5   |  2  |   -11.02871999 |     ?    |
  | 6   |  2  |   -11.02858537 |     ?    |
  | 7   |  2  |    -1.08915879 |     ?    |
  | 8   |  2  |    -0.95442037 |     ?    |
  | 9   |  2  |    -0.95438300 |     ?    |
  | 10  |  2  |    -0.76696711 |     ?    |
  | 11  |  2  |    -0.76696465 |     ?    |
  | 12  |  2  |    -0.66208322 |     ?    |
  | 13  |  2  |    -0.59530659 |     ?    |
  | 14  |  2  |    -0.55272543 |     ?    |
  | 15  |  2  |    -0.53418390 |     ?    |
  | 16  |  2  |    -0.53414641 |     ?    |
  | 17  |  2  |    -0.45547106 |     ?    |
  | 18  |  2  |    -0.43312876 |     ?    |
  | 19  |  2  |    -0.43302106 |     ?    |
  | 20  |  2  |    -0.27949448 |     ?    |
  | 21  |  2  |    -0.27948893 |     ?    |
  | 22  |  0  |     0.26832336 |     ?    |
  | 23  |  0  |     0.26834121 |     ?    |
  | 24  |  0  |     0.50273480 |     ?    |
  | 25  |  0  |     0.58075920 |     ?    |
  | 26  |  0  |     0.64982581 |     ?    |
  | 27  |  0  |     0.65013229 |     ?    |
  | 28  |  0  |     0.72109488 |     ?    |
  | 29  |  0  |     0.73392898 |     ?    |
  | 30  |  0  |     0.73418101 |     ?    |
  | 31  |  0  |     0.88478857 |     ?    |
  | 32  |  0  |     0.88480816 |     ?    |
  | 33  |  0  |     0.89573740 |     ?    |
  | 34  |  0  |     0.89578463 |     ?    |
  | 35  |  0  |     1.08471801 |     ?    |
  | 36  |  0  |     1.14918773 |     ?    |
  +-----+-----+----------------+----------+

        Mulliken population analysis
        ----------------------------

      Atom     Population     Charge
      1  C     6.06260860  -0.06260860
      2  C     6.06253300  -0.06253300
      3  C     6.06253300  -0.06253300
      4  C     6.06260860  -0.06260860
      5  C     6.06253300  -0.06253300
      6  C     6.06253300  -0.06253300
      7  H     0.93741271   0.06258729
      8  H     0.93745634   0.06254366
      9  H     0.93745634   0.06254366
      10 H     0.93741271   0.06258729
      11 H     0.93745634   0.06254366
      12 H     0.93745634   0.06254366


        Loewdin population analysis
        ---------------------------

      Atom     Population     Charge
      1  C     6.03051101  -0.03051101
      2  C     6.03044759  -0.03044759
      3  C     6.03044759  -0.03044759
      4  C     6.03051101  -0.03051101
      5  C     6.03044759  -0.03044759
      6  C     6.03044759  -0.03044759
      7  H     0.96951757   0.03048243
      8  H     0.96953812   0.03046188
      9  H     0.96953812   0.03046188
      10 H     0.96951757   0.03048243
      11 H     0.96953812   0.03046188
      12 H     0.96953812   0.03046188


		SCF properties
		--------------

  moments are calculated at point (0,0,0)

  dipole moment:
  D       x            y            z
    -0.00000000  -0.00000000   0.00000000
  |D| = 0.00000000 a.u. = 0.00000000 Debye

  second moments of charge (atomic units):
           xx           yy           zz
     -23.194510   -23.192592   -26.098328
           xy           xz           yz
      -0.000000     0.000000    -0.000000
  quadrupole moments (atomic units):
           xx           yy           zz
       2.901899     2.907655    -5.809554
           xy           xz           yz
      -0.000000     0.000000    -0.000000
  Q_ij = 3*r_i*r_j - \delta_ij*|r^2|
  1 Buckingham = 1.345044 a.u.


 time for (sec):
 -------------------------------------------------------
  One-electron integrals                          0.145
  Two-electron integrals                         26.033
  Basis set orthogonalization                     0.091
  Initial guess                                   0.039
  Fock matrix diagonalization                     0.043
  Density matrix construction                     0.002
  Fock matrix construction                        0.135
 -------------------------------------------------------
                                                 26.490


Allocated: 805912 bytes
Freed:     805912 bytes
Max usage: 207232 bytes
Total time: 26.486 sec
